### ISSUE TITLE
USA - 'Strengthen Pro-American Sentiment' operational success gives boosting Speer's Germany

### ISSUE THREAD

**Issue by Ikidikid** [TNODevs/TNO-Bug-Tracker#2355]
### Operating System

Windows 10 Pro

### HOI4 version

1.12.14

### TNO version

1.5.0

### Do you have other mods in use?

None

### Were you using Steam?

Yes

### Which major expansions do you NOT have?

I have all

### Files

[error.zip](https://github.com/TNODevs/TNO-Bug-Tracker/files/12810916/error.zip)
[game.zip](https://github.com/TNODevs/TNO-Bug-Tracker/files/12810917/game.zip)
[USA_play.zip](https://github.com/TNODevs/TNO-Bug-Tracker/files/12810922/USA_play.zip)

### Describe the bug

After selecting the 'Strengthen Pro-American Sentiment' operation in either the Central or South America region and the operational success, the operational success event gives PP and some stability to Speer's Germany instead of giving USA's PP. Probably cause the 'Radio Free Germany' operation failed previously.

### Steps to reproduce / Cause of the issue

1. Play as USA.
2. The German Civil War started.
3. Select the 'Look into the Reformer' focus.
4. Bill passed to support Speer during the GCW covertly.
5. One supporting Speer operational fails.
6. Select the 'Strengthen Pro-American Sentiment' operation.
7. Operational success.

### Expected behaviour

I don't know what causes this.

### Screenshots

![Image_1](https://github.com/TNODevs/TNO-Bug-Tracker/assets/131833099/c1d37bc4-8766-4919-85a7-91f6405ce1e9)
![Image_2](https://github.com/TNODevs/TNO-Bug-Tracker/assets/131833099/37947e9e-27cc-4145-8c15-ea84738157ae)
![Image_3](https://github.com/TNODevs/TNO-Bug-Tracker/assets/131833099/638c9870-0396-4c61-b6ad-4db6b626e4d5)

### Additional information

Not needed.

**Comment by Fortune801** [comment]
Can you explain what you believe the bug to be? If the US is supporting pro-American sentiments in Speer's Germany wouldn't it make sense for a successful operation to boost Speer's stability and political power?

**Comment by Ikidikid** [comment]
Oh, uh. The 'Radio Free Germany' operation failed previously before I took the 'Strengthen Pro-American Sentiment' operation. So, I believe this is a bug when the 'Radio Free Germany' operation failed.

**Comment by runelead00** [comment]
Think I figured it out, a bunch of missions were setting the flag that determine which mission you did outside the effect that decided success or failure, but those flags were only meant to be added and removed if the mission is successful, otherwise you get a generic failure event.

tl;dr Fixed in Dev Build
